volcanic-authenticator.gemspec

```diff
@@ -1,35 +1,26 @@
 # frozen_string_literal: true
 
-lib = File.expand_path('lib', __dir__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-# require_relative 'lib/volcanic/authenticator/version.rb'
+version = File.read(File.expand_path('VERSION', __dir__)).strip
```

Comment:
This means that you need to maintain the version in two places (the version file and in the code) which is brittle - at some point they will get our of sync.

Take a look at the cache gem's gemspec file to see a pattern which is preferred (i.e. just in the code, and load that version file)